package/system/signify/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=signify
+PKG_VERSION:=29
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/aperezdc/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=8d255ad88290c9b675711f47cc7336b736f6a7adcccd822a0c1a03e3d53de41c
+
+PKG_LICENSE:=ISC
+PKG_LICENSE_FILES:=
+
+PKG_MAINTAINER:=Paul Spooren <mail@aparcar.org>
+
+PKG_BUILD_PARALLEL:=1
+HOST_BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+HOST_BUILD_DEPENDS:=libbsd/host
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+
+TARGET_CFLAGS += -ffunction-sections -fdata-sections
+TARGET_LDFLAGS += -Wl,--gc-sections
+# MAKE_VARS += VERIFY_ONLY=1
```

Comment:
Is it necessary?

Reply:
@jow- suggested these lines via IRC to decrease the size, I don't know what they do :)

Reply:
The commented `VERIFY_ONLY` is not necessary in the form, more a TODO reminder to git it fixed as compiling failed for me. I'm working on it.